common/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -12,7 +12,9 @@ databaseChangeLog:
       file: db/changelog/v001/remove_not_null_job_output.sql
   - include:
       file: db/changelog/v001/ab2d-828-increase-username-column-length.sql
+  - include:
+      file: db/changelog/v001/ab2d-777-resolve_diff_ids.sql
```

Comment:
Does a new change-log (and hence the change-sets defined inside it) need to be appended at the end?
Not sure if it is okay to insert them in the midst of existing change-logs like this.

Reply:
I'm not sure what you mean. I've never used liquidbase before so I just went with what looked right and it worked.

Reply:
sounds good. 
I am used to seeing new change-logs being appended to the bottom of the list of existing change-logs and wasn't sure if adding it in the middle would cause any conflicts.  
Thanks for clarifying that it works fine.